.github/scripts/test_trymerge.py

```diff
@@ -17,6 +17,9 @@
 from unittest import TestCase, main, mock
 from urllib.error import HTTPError
 
+if 'GIT_REMOTE_URL' not in os.environ:
+    os.environ['GIT_REMOTE_URL'] = "https://github.com/pytorch/pytorch"
```

Comment:
nit, this can probably be done in one line, but overall lgtm

```suggestion
os.environ['GIT_REMOTE_URL'] = os.environ.get(GIT_REMOTE_URL, "https://github.com/pytorch/pytorch")
```

Reply:
I'm meh on this, it still requires hardcoding "GIT_REMOTE_URL" twice, and is less easy to read IMO